operator/src/main/resources/scripts/model_wdt_mii_filter.py

```diff
@@ -139,6 +139,13 @@ def toDNS1123Legal(self, address):
   def getModel(self):
     return self.model
 
+  def wlsVersionEarlierThan(self, version):
+    # unconventional import within function definition for unit testing
+    from weblogic.management.configuration import LegalHelper
+    # WLS Domain versions supported by operator are 12.2.1.3 + patches, 12.2.1.4
+    # and 14.1.1.0 so current version will only be one of these that are listed.
+    return LegalHelper.versionEarlierThan("14.1.1.0", version)
```

Comment:
The code that was removed by PR #2582 checks against 14.1.2.0. Just want to make sure that changing from 14.1.2.0 to 14.1.1.0 here is intentional.

Reply:
yes, it turns out from talking to Quang that secure mode will be enabled by default in 14.1.2.0 so had to change it.  This is intentional.